### PR TITLE
ZA2 | Xolani Mvana/Javascript-Core-1-Coursework-Week1

### PR DESCRIPTION
**Pull request by Xolani-M**
**Volunteers: Are you marking this coursework?** _You can find a guide on how to mark this coursework in `HOW_TO_MARK.md` in the root of this repository_

# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

# Notes

- What did you find easy?

- What did you find hard?

- What do you still not understand?

- Any other notes?

-----
[View rendered exercises/B-hello-world/README.md](https://github.com/Xolani-M/JavaScript-Core-1-Coursework-Week1/blob/master/exercises/B-hello-world/README.md)
[View rendered exercises/G-numbers/README.md](https://github.com/Xolani-M/JavaScript-Core-1-Coursework-Week1/blob/master/exercises/G-numbers/README.md)

### CODE REVIEW

File: exercises/G-numbers/exercise.js [hunk 1]
```diff
@@ -1 +1,6 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+let numberOfStudents = 15;
+let numberOfMentors = 8;
+console.log(`Number of students: ${numberOfStudents}`);
```
Comment (AdvocateM) [inline exercises/G-numbers/exercise.js:4]: Nice way of concatenating

**Review by AdvocateM — COMMENTED** [review]
Awesome!..

File: exercises/G-numbers/exercise.js (hunk 1 above)
Comment (Xolani-M) [reply exercises/G-numbers/exercise.js:4]: Hey Advocate,

Thanks bro.

**Review by riaanduplessis — COMMENTED** [review]
Awesome work Xolani

File: extra/1-currency-conversion.js [hunk 2]
```diff
@@ -15,7 +18,10 @@ function convertToUSD() {}
   They have also decided that they should add a 1% fee to all foreign transactions, which means you only convert 99% of the £ to BRL.
 */
 
-function convertToBRL() {}
+function convertToBRL(pound) {
+  let poundToBrl = pound * 5.643;
+  return poundToBrl.toFixed(2);
+}
```
Comment (riaanduplessis) [inline extra/1-currency-conversion.js:24]: This is good, you're almost there! Just look closely at the requirements again and make sure you're following the instructions 😉

**Pull request closed without merging**